### PR TITLE
ECO 5600

### PR DESCRIPTION
**Pull request by Raulkg**
- changes to make text smaller in landscape
- show "scroll for more" ticker as it is not possible to display scrollbars always
- scroll to top button implemented

### CODE REVIEW

**Review by jeffswartz — APPROVED** [review]
Please resolve the merge conflicts.

File: web/less/endMeeting.less [hunk 1]
```diff
@@ -107,4 +123,86 @@ html {
...
+.back-to-top-wrapper {
+  display:none;
+  position: absolute;
+  top: 100vh;
+    right: 3rem;
+    bottom: -22em;
+    width: 3em;
+  pointer-events: none;
+  @media @smartphonesPortrait {
+  display:block;
```
Comment (jeffswartz) [inline web/less/endMeeting.less:138]: Please indent. (But do we want this to be `display:block` in portrait mode?)

File: web/less/endMeeting.less [hunk 2]
```diff
@@ -107,4 +123,86 @@ html {
...
+  -webkit-transform: scale(1.1);
+          transform: scale(1.1);
+}
+.back-to-top-link:focus {
+  outline: none;
+  box-shadow: 0 0 0 3px #4e85c0;
+}
+
+.back-to-top-link::before {
+  content: "Scroll for more .. ";
```
[the remark this answers is not in the document]
Comment (Raulkg) [reply web/less/endMeeting.less:178]: it fades away after 6 seconds . is that okay?

File: web/less/endMeeting.less (hunk 1 above)
Comment (Raulkg) [reply web/less/endMeeting.less:138]: there is some content cropped in portrait mode too. helps the user identify scrollable content. WDYT?

**Pull request merged**